README.md

```diff
@@ -257,7 +257,14 @@ types are hashable as well, so you can construct stranger composite types
       StringIntegerMap(brian => 28, ian => 15, robey => 5000) => 0.2,
       StringIntegerMap(brian => 30, ian => 21, robey => 35) => 0.9)
 
+#### Choice Types ####
```

Comment:
I'm not quite sure where to put it; I've tried a change, but I'm open to suggestions about where it would scan better?